x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_weblogs/ml/low_count.json

```diff
@@ -0,0 +1,36 @@
+
+{
+  "groups": ["kibana_sample_logs", "kibana_sample_data"],
+  "description": "Kibana sample web logs data : find anomalies in the visitor count",
```

Comment:
As above, for consisency we should remove the space before the colon `:`

Reply:
yep - should be fixed now!